src/clr-angular/forms/datepicker/date-container.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component, OnDestroy} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {IfOpenService} from "../../utils/conditional/if-open.service";
+import {DynamicWrapper} from "../../utils/host-wrapping";
+import {FormControlService} from "../common/form-control.service";
+
+import {DayModel} from "./model/day.model";
+import {DateIOService} from "./providers/date-io.service";
+import {DateNavigationService} from "./providers/date-navigation.service";
+import {DatepickerEnabledService} from "./providers/datepicker-enabled.service";
+import {LocaleHelperService} from "./providers/locale-helper.service";
+
+@Component({
+    selector: "clr-date-container",
+    template: `
+        <ng-content></ng-content>
+        <button
```

Comment:
Note for later (don't solve it now): this button doesn't have a description for screenreaders.